doc/build-instructions/Build_Instructions_V8.md

```diff
@@ -99,19 +99,36 @@ bash mkdocker.sh --tag=openj9 --dist=ubuntu --version=16.04 --print
 ```
 apt-get update \
  && apt-get install -qq -y --no-install-recommends \
-   autoconf \
-   ca-certificates \
    ...
 ```
 
-2. Download and setup **freemarker.jar** into a directory. The example commands use `/root` to be consistent with the Docker instructions. If you aren't using Docker, you probably want to store the **freemarker.jar** in your home directory.
+2. The previous step installed g++-7 and gcc-7 packages, which might be different
+than the default version installed on your system. Export variables to set the 
+version used in the build.
+```
+export CC=gcc-7 CXX=g++-7
+```
+
+3. Download and setup **freemarker.jar** into a directory. The example commands use `/root` to be consistent with the Docker instructions. If you aren't using Docker, you probably want to store the **freemarker.jar** in your home directory.
 ```
 cd /root
 wget https://sourceforge.net/projects/freemarker/files/freemarker/2.3.8/freemarker-2.3.8.tar.gz/download -O freemarker.tgz
 tar -xzf freemarker.tgz freemarker-2.3.8/lib/freemarker.jar --strip=2
 rm -f freemarker.tgz
 ```
 
+4. Download and setup the boot JDK using the latest AdoptOpenJDK v8 build.
+```
+cd /<my_home_dir>
+wget -O bootjdk8.tar.gz "https://api.adoptopenjdk.net/v3/binary/latest/8/ga/linux/x64/jdk/openj9/normal/adoptopenjdk"
+tar -xzf bootjdk8.tar.gz
+rm -f bootjdk8.tar.gz
+mv $(ls | grep -i jdk) bootjdk8
+
+export JAVA_HOME="/<my_home_dir>/bootjdk8"
+export PATH="${JAVA_HOME}/bin:${PATH}"
```

Comment:
This can be removed given we're recommending use of `--with-boot-jdk`.

Reply:
Pushed another update.